tasks/main.yml

```diff
@@ -44,6 +44,7 @@
   get_url:
     url: "http://{{ jenkins_hostname }}:{{ jenkins_http_port }}{{ jenkins_url_prefix }}/jnlpJars/jenkins-cli.jar"
     dest: "{{ jenkins_jar_location }}"
+    use_proxy: no
```

Comment:
@zshi-redhat - It seems like this could possibly break the role for certain users (e.g. if they _must_ use a proxy to download the file).

I would be willing to have a variable for this (defaulting to `yes` since that's the default in Ansible's `get_url` module), but I won't merge it as is.